### DISCUSSION TITLE
Proxy Support CrowdFund for Feature

### DISCUSSION THREAD

**Discussion by Jmanfrom87** [MalloyDelacroix/DownloaderForReddit#381] [Ideas]
Anybody Interested in pooling funds together in order to get rotating proxy support?

Redgifs bans you pretty quickly from downloading Gifs. We need to have rotating proxy support so that you can upload a list of a few 100 proxies and rotate them one at a time for each download request so you dont get banned or rate limited.

I reached out to a few Python developers and I have been quoted about $500 bucks to fork this program and build in rotating proxies. If you are interested in pooling funds together to get this done, respond here.

**Comment by MalloyDelacroix** [comment]
In my opinion, the rate limits that reddit imposed earlier this year have made this application's future a little hazy.  With the inconsistent connection issues to reddit, I wouldn't recommend spending that amount of money to circumvent what redgifs is doing.  They also have changed their API many times in the last year.
